### ISSUE TITLE
Outdated dependencies for package.json in Image Service

### ISSUE THREAD

**Issue by cindyledev** [Seneca-CDOT/telescope#2438]
Related #2387 #2389 #2404 #2437

// src/api/image/package.json
```
Package                Current  Wanted  Latest  Location                            Depended by
@senecacdot/satellite   1.16.0  1.17.0  1.17.0  node_modules/@senecacdot/satellite  image
nodemon                  2.0.7  2.0.14  2.0.14  node_modules/nodemon                image
sharp                   0.29.1  0.29.2  0.29.2  node_modules/sharp                  image
```

**Comment by JiaHua-Zou** [comment]
Can you assign this to me?

**Comment by cindyledev** [comment]
Yes, there have been minor updates for the nodemon and sharp packages so please update accordingly

```
$ npm outdated
Package                Current  Wanted  Latest  Location                            Depended by
@senecacdot/satellite   1.16.0  1.17.0  1.17.0  node_modules/@senecacdot/satellite  image
nodemon                  2.0.7  2.0.15  2.0.15  node_modules/nodemon                image
sharp                   0.29.1  0.29.3  0.29.3  node_modules/sharp                  image
```

**Comment by humphd** [comment]
@JiaHua-Zou all yours

**Comment by humphd** [comment]
Fixed by #2483.  @JiaHua-Zou next time uses "Fixes #..." in your PR description so it automatically closes the associated issue.
